src/helpers.js

```diff
@@ -225,3 +225,22 @@ export function getFormatedDateTime (timestamp) {
     date: `${t.getDate()}/${t.getMonth()}/${t.getFullYear()}`,
   }
 }
+
+export function formatDateFromNow (d) {
```

Comment:
Would be great to have a few unit tests for this function so:
  1. its output can be stable, and
  2. we can see, at a glance, what the output should be based on various inputs (sometimes it is easier to figure out what a function does by looking at its tests)